src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -0,0 +1,130 @@
+// Data attributed to https://iextrading.com/api-exhibit-a
+
+const ApiConsumer = require('../utils/api-consumer');
+
+
+let StockConsumer = new ApiConsumer('iex-trading', 'https://api.iextrading.com/1.0',{cache: {ttl: 3600*24*30*6}});
+
+const queryOpts = {
+    queryString: '/stock/',
+    headers:{
+    },
+    body: undefined,
+    cache: true
+};
+
+let stockParser = resp => {
+    let results = [];
+    results = resp[company];
+	    
+    return results;
+};
+
+/**
+ * Get the selling price of the specified company, with a 15 min delay
+ * @param {companySymbol} Symbol. The stock ticker symbol of the desired company
+ * @returns {double} Value of the current price for that stock
+ */
+StockConsumer.currentPrice = function(companySymbol) {
+	// Defines the query options for the API, including the user-defined value to receive
+    queryOpts.queryString = '';
+    queryOpts.queryString = '/stock/' + companySymbol + '/quote';
+
+    let company = 'latestPrice';
```

Comment:
I would either rename this variable to `field` or just change line 38 to `return body.latestPrice;`

Reply:
I would also update this in the rest of the RPCs as well